src/containers/add-attachments/AddAttachments.constants.tsx

```diff
@@ -0,0 +1,34 @@
+import { Attachment, RemoveColumnRules, SortEnum } from '~/types'
+
+import { getFormattedDate } from '~/utils/helper-functions'
+import { convertBytesToProperSize } from '~/utils/bytes-to-proper-size'
+
+import IconExtentionWithTitle from '~/components/icon-extention-with-title/IconExtentionWithTitle'
```

Comment:
import IconExtentionWithTitle from '~/components/icon-extention-with-title/IconExtentionWithTitle'

import { getFormattedDate } from '~/utils/helper-functions'
import { convertBytesToProperSize } from '~/utils/bytes-to-proper-size'
import { Attachment, RemoveColumnRules, SortEnum } from '~/types'

Reply:
done